packages/core/src/components/spinner/_spinner.scss

```diff
@@ -16,7 +16,7 @@
   justify-content: center;
 
   // allow paths to overflow container -- critical for edges of circles!
-  overflow: visible;
+  overflow: visible hidden;
```

Comment:
any ideas why https://github.com/palantir/blueprint/commit/21db581d9adac5380ed3c77aa9266e639481f330#diff-746703d417092b84b1146be9dc09d0810c0eb37a08d0a867d2f319c197d2e297R32 doesn't apply?

if this is a blind fix because it seems to work I'd prefer if this is fixed where used - ex you may be setting scroll on your parent wrapper for the content that loads after but can hide overflow yourself while loading.

Reply:
got it, closing this